### PR TITLE
docs: Added warning about Animate.css version migration  #7602

### LINKED ISSUE #7602: [docs] Animations page should mention migrating from 3.X of animate.css to 4.
Since Quasar's Animations is mainly borrowed from **animated.css** which has a breaking change:
Starting from version 4, all CSS classes need to be prefixed with `animate__ `  

Although they also provide the "animate.compat.css" version, I think the migration should be mentioned in the Animations Page of the Quasar.

ref: [Migration from v3.x and under](https://animate.style/#migration)

### PR DESCRIPTION
**Pull request by kevin61416**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [x] Documentation
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch and _not_ the `master` branch
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] It's been tested on a Cordova (iOS, Android) app
- [ ] It's been tested on a Electron app
- [ ] Any necessary documentation has been added or updated [in the docs](https://github.com/quasarframework/quasar/tree/dev/docs) (for faster update click on "Suggest an edit on GitHub" at bottom of page) or explained in the PR's description.

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

File: docs/src/pages/options/animations.md [hunk 1]
```diff
@@ -37,6 +37,10 @@ If you are building a website, you can also skip configuring quasar.conf.js and
 </head>
 ```
 
+::: warning
+It should be noted that when you import Animate.css through the `<link>` tag, all animation CSS classes must be prefixed with `animate__`. This is a breaking change in the migration of Animate.css from v3 to v4. If you want to avoid using prefixes, you can import [compact version](https://animate.style/#migration).
```
Comment (hawkeye64) [inline docs/src/pages/options/animations.md:41]: Change `you can import [compact version]` to `you can import the [compact version]`

Comment (hawkeye64) [reply docs/src/pages/options/animations.md:41]: Also, make a note that if using Quasar, then no additional changes are needed.

Comment (kevin61416) [reply docs/src/pages/options/animations.md]: OK, the modification is complete, thank you 😀

**Comment by pdanpdan** [comment]
I think it should be compat version instead of compact version

**Comment by kevin61416** [comment]
sorry for my weak eyes...

**Comment by rstoenescu** [comment]
Thanks for contributing!

**Pull request merged**